<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSystemAdmin</class>
 <widget class="QDialog" name="AddSystemAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 144, 96);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 500 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string> The Rose Garden System  |      User Access Control</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>531</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font: 500 12pt &quot;Ubuntu&quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>User ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>User Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_UserName">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Password">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewAccessDetail">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 251, 232);
font: 500 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>View Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertAccessDetail">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 95, 180);
font: 500 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteAccessDetail">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);
font: 500 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="lableID">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>ID *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_UserName">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>User name *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>Password *</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
